sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/TracerProviderConfiguration.java

```diff
@@ -85,17 +85,17 @@ static SpanLimits configureSpanLimits(ConfigProperties config) {
 
     Integer maxAttrs = config.getInt("otel.span.attribute.count.limit");
     if (maxAttrs != null) {
-      builder.setMaxNumberOfAttributes(maxAttrs);
+      builder.setSpanAttributeLimit(maxAttrs);
```

Comment:
I'd vote for `setAttributeCountLimit()` etc in order to match the env var naming (and agree with leaving off prefix otel.span as redundant)